docs/developers/release.md

```diff
@@ -4,10 +4,47 @@ This guide documents `napari`'s release process.
 Currently, it only handles distribution, but as the project matures,
 it will include generating release notes, documentation, etc.
 
-This is mainly meant for the core developers who will actually be performing the release.
-They will need to have a [PyPI](https://pypi.org) account with upload permissions to the `napari` package.
+# Timeline
+New versions of `napari` will be released every two months. The first release candidate will be available one week prior to release for testing purposes. Multiple release candidates may become available during the week prior to release.
 
-You will also need the additional `release` dependencies (`pip install -e .[release]`) to complete the release process.
+The latest release candidate can be installed with 
+
+`pip install --pre napari`
+
+# Release management
+The release will be coordinated by a release manager whose responsibilities include...
+
+## Two weeks before release (one week before release candidate)
+- Look through currently open PRs and get a sense of what would be good to merge before the first release candidate 
+- Create a zulip thread in the release channel letting people know the release candidate is coming and pointing out PRs that would be nice to merge before release
+
+At this stage, bug fixes should and features that are close to landing should be prioritised. The release manager will follow up with PR authors, reviewing and merging as needed.
```

Comment:
```suggestion
At this stage, bug fixes and features that are close to landing should be prioritized. The release manager will follow up with PR authors, reviewing and merging as needed.
```

I think we use `z`s and skip out on `u`s around here 😆 Just kidding but to be consistent we should use US spelling across the board